doc/source/security/data-at-rest-encryption.rst

```diff
@@ -152,6 +156,7 @@ The following is a configuration file example: ::
 
   vault_url = https://vault.public.com:8202
   secret_mount_point = secret
+  secret_mount_point_version = 1
```

Comment:
Lat's make the default value in the example to be `AUTO` - less chances of something going wrong for the user
```suggestion
  secret_mount_point_version = AUTO
```

Reply:
Updated.